apps/web/pages/my-account/personal-data.vue

```diff
@@ -1,7 +1,79 @@
-<template>{{ $t('account.accountSettings.section.personalData') }}</template>
+<template>
+  <UiDivider class="col-span-3 w-screen md:w-auto -mx-4 md:mx-0" />
+  <AccountData
+    class="col-span-3"
+    :header="$t('account.accountSettings.personalData.yourName')"
+    :button-text="$t('account.accountSettings.personalData.edit')"
+    @on-click="openModal('name')"
+  >
+    {{ userData.firstName }} {{ userData.lastName }}
+  </AccountData>
+  <UiDivider class="col-span-3 w-screen md:w-auto -mx-4 md:mx-0" />
+  <AccountData
+    class="col-span-3"
+    :header="$t('account.accountSettings.personalData.contactInformation')"
+    :button-text="$t('account.accountSettings.personalData.edit')"
+    @on-click="openModal('mail')"
+  >
+    {{ userData.email }}
+  </AccountData>
+  <UiDivider class="col-span-3 w-screen md:w-auto -mx-4 md:mx-0" />
+  <AccountData
+    class="col-span-3"
+    :header="$t('account.accountSettings.personalData.yourPassword')"
+    :button-text="$t('account.accountSettings.personalData.change')"
+    @on-click="openModal('password')"
+  >
+    {{ userData.password }}
+  </AccountData>
+  <UiDivider class="col-span-3 w-screen md:w-auto -mx-4 md:mx-0" />
+  <UiOverlay v-if="isOpen" :visible="isOpen">
+    <SfModal
+      v-model="isOpen"
+      as="section"
+      role="dialog"
+      class="h-full w-full overflow-auto md:w-[600px] md:h-fit"
+      aria-labelledby="address-modal-title"
+    >
+      <header>
+        <SfButton square variant="tertiary" class="absolute right-2 top-2" @click="close">
+          <SfIconClose />
+        </SfButton>
+        <h3 id="address-modal-title" class="text-neutral-900 text-lg md:text-2xl font-bold mb-6">
+          {{
+            openedForm === 'name'
+              ? $t('account.accountSettings.personalData.yourName')
+              : openedForm === 'mail'
+              ? $t('account.accountSettings.personalData.contactInformation')
+              : $t('account.accountSettings.personalData.passwordChange')
+          }}
```

Comment:
suggestion: imho we should use parts of already existing strings
```suggestion
          {{ $t(`account.accountSettings.personalData.${openedForm}`) }}
```
and then just `openModal('yourName')` etc